src/hooks/useAuditLogsOptimized.ts

```
@@ -0,0 +1,297 @@
+      if (filters.entityId) {
+        query = query.eq('entity_id', filters.entityId);
+      }
+
+      if (filters.dateRange) {
+        query = query
+          .gte('created_at', filters.dateRange.start.toISOString())
+          .lte('created_at', filters.dateRange.end.toISOString());
+      }
+
+      // Search in details JSON if search term provided (uses GIN index)
+      if (filters.searchTerm && filters.searchTerm.trim() && includeDetails) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)